packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -463,7 +499,7 @@ class SystemRulesTable extends React.Component {
                         onSort={ this.onSort }
                         sortBy={ sortBy }
                         onSelect={ this.onSelect }
```

Comment:
You should add `onSelect={ (currentRows.length !== 0) && this.onSelect }` so you wouldn't see that checkbox when table is empty.

Reply:
Nice, we have had that in our backlog for a while but wasn't sure how to get rid of that, thanks!